versioned/persist/dynamodb/src/main/java/org/projectnessie/versioned/persist/dynamodb/DynamoDatabaseAdapter.java

```diff
@@ -127,11 +128,11 @@ static RuntimeException unhandledException(String operation, RuntimeException e)
 
   @Override
   public void eraseRepo() {
-    client.client.deleteItem(
-        b -> b.tableName(TABLE_GLOBAL_POINTER).key(globalPointerKeyMap).build());
+    client.client.deleteItem(b -> b.tableName(TABLE_GLOBAL_POINTER).key(globalPointerKeyMap));
 
     try (BatchDelete batchDelete = new BatchDelete()) {
-      Stream.of(TABLE_GLOBAL_LOG, TABLE_COMMIT_LOG, TABLE_KEY_LISTS, TABLE_REF_LOG)
+      allExceptGlobalPointer()
```

Comment:
Why are we not erasing all tables (perhaps I forgot :facepalm: )?

Reply:
global-pointer's a little bit different (one line above)